osu.Server.Queues.ScoreStatisticsProcessor/Commands/Queue/BatchInserter.cs

```diff
@@ -315,40 +262,36 @@ public static async Task<ScoreInfo> CreateReferenceScore(Ruleset ruleset, HighSc
             // A special hit result is used to pad out the combo value to match, based on the max combo from the beatmap attributes.
             int maxComboFromStatistics = scoreInfo.MaximumStatistics.Where(kvp => kvp.Key.AffectsCombo()).Select(kvp => kvp.Value).DefaultIfEmpty(0).Sum();
 
-            BeatmapScoringAttributes? scoreAttributes = await connection.QuerySingleOrDefaultAsync<BeatmapScoringAttributes>(
-                "SELECT * FROM osu_beatmap_scoring_attribs WHERE beatmap_id = @BeatmapId AND mode = @RulesetId", new
-                {
-                    BeatmapId = highScore.beatmap_id,
-                    RulesetId = ruleset.RulesetInfo.OnlineID
-                }, transaction);
+            var scoreAttributes = getScoringAttributes(rulesetId, highScore.beatmap_id).Result;
 
             if (scoreAttributes == null)
             {
                 // TODO: LOG
-                await Console.Error.WriteLineAsync($"{highScore.score_id}: Scoring attribs entry missing for beatmap {highScore.beatmap_id}.");
+                Console.Error.WriteLineAsync($"{highScore.score_id}: Scoring attribs entry missing for beatmap {highScore.beatmap_id}.");
```

Comment:
```suggestion
                Console.Error.WriteLine(highScore.score_id}: Scoring attribs entry missing for beatmap {highScore.beatmap_id}.");
```

weird there's no inspection for this